app/view/gallery/gallery.html

```diff
@@ -0,0 +1,3 @@
+<section class="gallery-content">
+  <h2>The Gallery!</h2>
```

Comment:
It would be best practice to abstract the title out into the GalleryController and then get the title off of the controller and render it here 👍